includes/admin/forms/dashboard-columns.php

```diff
@@ -404,8 +404,8 @@ function give_form_search_query_filter( $wp ) {
 
 		$wp->query_vars['date_query'] =
 			array(
-				'after'     => ! empty( $_GET['start-date'] ) ? give_get_formatted_date( $_GET['start-date'] ) : false,
```

Comment:
What is the recommended usage of `give_get_formatted_date()` now?

Reply:
@DevinWalker This function does not work fine with translated localize date strings.
I updated the function description:

https://github.com/impress-org/give/commit/b8edffce278e300d6ff201aef1e095b5c9469c66